test/test_workflow.cpp

```diff
@@ -553,6 +553,73 @@ TEST(SteadyNS_Workflow, ComponentSeparateVariable)
    return;
 }
 
+TEST(LinElast_Workflow, MFEMIndividualTest)
+{
+   config = InputParser("inputs/linelast.base.yml");
+
+   config.dict_["model_reduction"]["rom_handler_type"] = "mfem";
+   config.dict_["model_reduction"]["visualization"]["enabled"] = true;
+   config.dict_["model_reduction"]["visualization"]["prefix"] = "basis_paraview";
+   config.dict_["sample_generation"]["parameters"][0]["sample_size"] = 2;
```

Comment:
We don't really make so many samples here, but can we increase the sample size a bit, say 4?